.pre-commit-config.yaml

```diff
@@ -40,14 +40,14 @@ repos:
   hooks:
     - id: check-github-workflows
 
-- repo: https://github.com/ansys/pre-commit-hooks
-  rev: v0.2.9
-  hooks:
-  - id: add-license-headers
-    args:
-    - --start_year=2023
-    exclude: |
-        (?x)^(
-            tests/test_reuse_files/bad_chars.py |
-            tests/test_reuse_files/index_error.scss
-        )$
\ No newline at end of file
+# - repo: https://github.com/ansys/pre-commit-hooks
+#   rev: v0.2.9
+#   hooks:
+#   - id: add-license-headers
+#     args:
+#     - --start_year=2023
+#     exclude: |
+#         (?x)^(
+#             tests/test_reuse_files/bad_chars.py |
+#             tests/test_reuse_files/index_error.scss
+#         )$
```

Comment:
Shouldn't this be reactivated eventually?

Reply:
@RobPasMue yes, I thought that this would conflict with the current repo since I updated the reuse version? But now that I'm thinking about it, it probably wouldn't get messed up right?